src/pages/Subscription/Subscription.jsx

```diff
@@ -11,6 +11,14 @@ export default function SubscriptionPage() {
     window.alert(inputState);
   }
 
+  function handleRevertButton() {
+    window.alert(revertButton(inputState));
+  }
+
+  function revertButton(s) {

```

Comment:
When you're defining a name of a function, you just need to tell what it does. 
In this case you're reverting a text so instead of `revertButton` you're `revertText`